assets/js/common/ModifiedCheckPill/ModifiedCheckPill.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import Pill from '@common/Pill';
+
+function ModifiedCheckPill({ className, customized = false }) {
```

Comment:
What about adding a story?

Reply:
A specific value does not add much value, imho, as it is just a pill (already has stories) and is mainly showcased where used.